### PR TITLE
add my handwired split RGB keyboard

### PR DESCRIPTION
**Pull request by digex**
## Description

my split RGB keyboard

RGB matrix

stmf103c8t6

20240716_171712-EDIT

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Please refrain from opening new PRs for the same content.

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/chconf.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#define CH_CFG_ST_TIMEDELTA 0
+
+#define CH_CFG_USE_CONDVARS_TIMEOUT FALSE
+
+#include_next <chconf.h>
+
+
+
+#define _CHIBIOS_RT_CONF_
+#define _CHIBIOS_RT_CONF_VER_6_0_
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/chconf.h:30]:
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config.h
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config.h:1]: Most of the content here can be migrated to json, and commented out code can/should be removed.

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config.h [hunk 2]
```diff
@@ -0,0 +1,210 @@
+/* Copyright 2020 petrvden */
+
+#pragma once
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config.h:5]:
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config_common.h
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/config_common.h:1]: Remove this.

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/halconf.h [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#define HAL_USE_PWM TRUE
+
+#define HAL_USE_SERIAL TRUE
+
+#define PAL_USE_CALLBACKS TRUE
+
+#define _CHIBIOS_HAL_CONF_
+#define _CHIBIOS_HAL_CONF_VER_7_0_
+
+#include "mcuconf.h"
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/halconf.h:30]:
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    "maintainer": "petrvden",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "processor": "STM32F103",
+    "url": "",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+       // "tap_dance" : true,
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json:13]: Comments should be removed. 
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+        "extrakey": true,
+        "mousekey": true,
+        //"nkro": true,
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json:16]:
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json [hunk 6]
```diff
@@ -0,0 +1,285 @@
+{
+    "manufacturer": "Petrov Denis",
+    "keyboard_name": "ortho_split_rgb_matrix_keyboard",
+    "maintainer": "petrvden",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "processor": "STM32F103",
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/info.json:7]: So, a bluepill? 
```suggestion
    "development_board": "bluepill",
    "diode_direction": "ROW2COL",
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/keymaps/default/keymap.c:1]: default keymap should be pristine, and simple.

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/mcuconf.h [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+/*
+ * USB driver system settings.
+ */
+#define STM32_USB_USE_USB1                  TRUE
+#define STM32_USB_LOW_POWER_ON_SUSPEND      FALSE
+#define STM32_USB_USB1_HP_IRQ_PRIORITY      13
+#define STM32_USB_USB1_LP_IRQ_PRIORITY      14
+
+
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/mcuconf.h:30]:
```suggestion
```

File: keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/ortho_split_rgb_matrix_keyboard.c [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+#endif
```
Comment (drashna) [inline keyboards/handwired/digex/ortho_split_rgb_matrix_keyboard/ortho_split_rgb_matrix_keyboard.c:49]:
```suggestion
void board_init(void) {
    AFIO->MAPR |= AFIO_MAPR_USART1_REMAP | AFIO_MAPR_TIM3_REMAP_PARTIALREMAP;
}
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**